src/site/content/en/fast/signed-exchanges-webpackager/index.md

```diff
@@ -161,7 +142,7 @@ used with signed exchanges.
 
```

Comment:
In step 3 above, I think you need `./webpkgserver` instead of `webpkgserver`. `go build` just builds the binary into the current working directory ([doc](https://golang.org/cmd/go/#hdr-Compile_packages_and_dependencies)).

(Alternatively, `go install` ([doc](https://golang.org/cmd/go/#hdr-Compile_and_install_packages_and_dependencies)) will install the binary into `~/${GOPATH-go}/bin`, but then we'll need instructions to add that dir to `$PATH`.)